uw-research-computing/gpu-jobs.md

```diff
@@ -223,12 +144,28 @@ attributes shown above. If you want a certain class of GPU, use CUDACapability:
 
 - **Specify Multiple Requirements (optional)**: Multiple requirements can be specified by using && statements:
 	```
-	requirements = (CUDACapability >= 7.5) && (CUDAGlobalMemoryMb >= 5000)
+requirements = (CUDACapability >= 7.5) && (CUDAGlobalMemoryMb >= 5000)
 	```
 	{:.sub}
 	Ensure all specified requirements match the attributes of the GPU/Server of interest. HTCondor matches jobs to GPUs that match all specified requirements. Otherwise, the jobs will sit idle indefinitely.
- 
-A complete sample submit file is shown below. There are also example submit files and 
+
+- **Indicate Software or Data Requirements**: If your data is large enough to 
+	use our `/staging` data system (see more information [here](file-avail-largedata.html)), 
+	or you are using modules or other software in our shared `/software` system, include 
+	the needed requirements (combining with any other GPU requirements as shown above). 
+
+- **Indicate Short/Resumable Jobs**: if your jobs are shorter than 4-6 hours, or have 
```

Comment:
We already use "short" as a value for `+GPUJobLength` and may want to distinguish that from this option.  Also this feels slightly redundant with the "Access Research Group GPUs" section.

Reply:
I tried to clean this up a bit. Let me know what you think.